Can't anymore use nuxt as devDependencies, because nuxt-start is missing vue-client-only

### Version

[v2.9.1](https://github.com/nuxt.js/releases/tag/v2.9.1)

### Reproduction link

[http://none](http://none)

### Steps to reproduce

add `nuxt`  to the  _devDependencies_ `npm i -D nuxt` 
add `nuxt-start` to the _dependencies_ `npm i nuxt-start`
build the app `npx nuxt build`
run  `npm ci --production` for production dependencies
run nuxt: `npx nuxt-start` 

### What is expected ?

Nuxt to run normally 

### What is actually happening?

```bash
ERROR  Cannot find module 'vue-client-only'

  at Function.module.exports [as sync] (node_modules/resolve/lib/sync.js:74:15)
  at node_modules/vue-server-renderer/build.prod.js:1:77711
  at Object.<anonymous> (server.js:1:0)
  at r (server.js:1:0)
  at Module.<anonymous> (server.js:1:0)
  at r (server.js:1:0)
  at Object.<anonymous> (server.js:1:0)
  at r (server.js:1:0)
  at server.js:1:1432
  at Object.<anonymous> (server.js:1:0)
```

### Additional comments?

Before Nuxt 2.9 this was  possible to do this without any problems

<!--cmty--><!--cmty_prevent_hook-->
<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c9667">#c9667</a>)</em></sub></div>

--- a/distributions/nuxt-start/package.json
+++ b/distributions/nuxt-start/package.json
@@ -59,6 +59,7 @@
     "@nuxt/core": "2.9.1",
     "node-fetch": "^2.6.0",
     "vue": "^2.6.10",
+    "vue-client-only": "^2.0.0",
     "vue-meta": "^2.2.1",
     "vue-no-ssr": "^1.1.1",
     "vue-router": "~3.0.7",

